### ISSUE TITLE
summarised `flutter doctor` output given by `flutter create` is grammatically dubious

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#6304]
The summarised `flutter doctor` output given by `flutter create` is grammatically dubious:

```
[✓] Flutter is fully installed. (on Mac OS, channel master)
[✓] Android toolchain - develop for Android devices is fully installed. (Android SDK 24.0.3)
[-] iOS toolchain - develop for iOS devices is partially installed; more components are available.
[x] Flutter IDE Support is not installed. (No supported IDEs installed)
[✓] Connected devices is fully installed.
```

Only the first of these actually makes sense. The others are quite confusing.

**Comment by sethladd** [comment]
I might shorten the iOS line to "iOS toolchain - missing critical components"

As I understand it , you would not be able to develop for iOS given your current state.

**Comment by mit-mit** [comment]
Do we need those suffixes on the toolchain ones? Can we just say `Android toolchain` rather than `Android toolchain - develop for Android devices`? It should be pretty clear what those are for.

**Comment by dnfield** [comment]
This was fixed in #19676
